samples/node/shared_subscription/index.ts

```diff
@@ -159,7 +159,7 @@ async function runSample(args : any) {
     }
 
     // If this is CI, append a UUID to the topic
-    if (input_isCi) {
+    if (input_isCi == true) {
         input_topic += "/" + Math.floor(Math.random() * 100000000);
```

Comment:
Could we make it a default user case? If the user does not specify the topic name, we always use a "test_topic + random_id" as the default topic string?

Reply:
We could, but then we'd require the user to have an IAM policy with a wildcard at the end if they run the sample without a topic. For the documentation, we say the topic is `test/topic` and not `test/topic/<uuid>` currently.

We could change it in the future though, but right now with the shared subscription release, I think we should leave it for now. I'll add a note though to revisit this when I go through the samples again as part of the cmdUtils stuff.

Reply:
For `is_ci`, maybe we could remove it and use an environment variable? I dunno...
For now I'll merge the PR since we use `is_ci` in several places, changing it can be a future PR.